### DISCUSSION TITLE
know function bot chat is used

### DISCUSSION THREAD

**Discussion by chaitanyakothakota** [Roshankrsoni/Roshankrsoni.github.io#6] [Q&A]
i know function bot chat is used and how to used and can you provide code of online bot chat.

**Comment by codenamekii** [comment]
Hi 👋,

The chatbot used on this website is likely a third-party service like **Tidio**, **Chatra**, or similar — usually embedded with a simple script tag.

If you're looking to implement an **online chatbot** in your own site, here’s a basic example using [Tidio](https://www.tidio.com/), which is free and easy to integrate.

### ✅ Steps to Add Tidio Chatbot:
1. Sign up at https://www.tidio.com/
2. Go to **Channels > Live Chat** and copy the integration script
3. Paste it in your HTML/Next.js file

### 💻 Example (HTML):
```html
<script src="//code.tidio.co/your-tidio-key.js" async></script>


or For Next.js, you can use next/head:

import Head from 'next/head';

export default function Home() {
  return (
    <>
      <Head>
        <script src="//code.tidio.co/your-tidio-key.js" async></script>
      </Head>
      <main>...</main>
    </>
  );
}


if you want to build your own, you can also use: 
1. Dialogflow
2. Botpress
3. Rasa

Let me know if you'd like a full example using custom chatbot logic with backend integration!

Hope this helps!
